### ISSUE TITLE
sorry to bother you again, when i execute the command of "CUDA_VISIBLE_DEVICES=0  python3 -m train.train_linemod_pvn3d --cls cat", the following error occured:

### ISSUE THREAD

**Issue by tiexuedanxin** [ethnhe/PVN3D#6]

**Comment by tiexuedanxin** [comment]
Traceback (most recent call last):
  File "/mnt/data/heyong/anaconda3/envs/PVN3D/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/mnt/data/heyong/anaconda3/envs/PVN3D/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/mnt/data/heyong/PycharmProjects/PVN3D-master/pvn3d/train/train_linemod_pvn3d.py", line 545, in <module>
    mode = 'triangular'
  File "/mnt/data/heyong/PycharmProjects/PVN3D-master/pvn3d/lib/utils/warmup_scheduler/scheduler.py", line 242, in __init__
    super().__init__(optimizer)
  File "/mnt/data/heyong/anaconda3/envs/PVN3D/lib/python3.6/site-packages/torch/optim/lr_scheduler.py", line 77, in __init__
    self.step()
TypeError: step() missing 1 required positional argument: 'step'
==============================================================

I find the definition of self.step() is def step(self, epoch=None): in the File "/mnt/data/heyong/anaconda3/envs/PVN3D/lib/python3.6/site-packages/torch/optim/lr_scheduler.py", the error means missing the parameter of epoch.
I think it mabey the python3.6 version cause the problem, could you tell me the python version of you or give me some advice to solve the problem, thank you for advance.

**Comment by ethnhe** [comment]
It may due to the torch version. The LRScheduler has something changed since torch version 1.1.0.
One way to solve the problem is to:
```
# Downgrade torch version to 1.0.1
pip3 install torch==1.0.1.post2 torchvision==0.2.0 --user

# rebuild PointNet++
cd PVN3D
rm -r build # if exists
python3 setup.py build_ext
```

**Comment by LounesDev** [comment]
But PyTorch 1.0.1.post2 isn't available for CUDA 10.0, is it ?

**Comment by kaixin-bai** [comment]
This version works fine for me.
`conda install pytorch==1.0.1 torchvision==0.2.2 cudatoolkit=10.0 -c pytorch`
